ProcessLib/LiquidFlow/LiquidFlowProcess.h

```diff
@@ -72,6 +77,14 @@ class LiquidFlowProcess final : public Process
                                           GlobalVector const& x) override;
 
     bool isLinear() const override { return true; }
+
+    std::vector<MaterialLib::Fluid::FluidProperties*> getFluidProperties() const
+    {
+        std::vector<MaterialLib::Fluid::FluidProperties*> material_properties =
+            {_material_properties->getFluidProperties()};
```

Comment:
I could not find the place where `getFluidProperties()` is used? Why is it necessary to return a `std::vector<MaterialLib::Fluid::FluidProperties*>` and not only something like `MaterialLib::Fluid::FluidProperties const&`?